salt/modules/nacl.py

```diff
@@ -183,12 +184,13 @@ def _get_config(**kwargs):
     '''
     Return configuration
     '''
+    cert_path = r'C:\salt\conf\pki\master' if sys.platform == 'win32' else '/etc/salt/pki/master/'
```

Comment:
Rather than hard coding these here, is the configured path value available through `__opts__['pki_dir']`?

Reply:
how so?

Reply:
Literally. 😄 

I'd probably just change the code to this form:

```
'sk_file': os.path.join(__opts__['pki_dir'], 'nacl'),
```

and remove the `cert_path` variable as it is unneeded.

Reply:
ok, i'll look into it. Still getting familiar with the code.